about.html

```diff
@@ -5,99 +5,131 @@
 <head>
   <meta charset="utf-8">
 
-  <title>Sprint Challenge - Home</title>
+  <title>Sprint Challenge - About</title>
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
 
 </head>
 
 <body>
-  <div class="container">
-
-    <!--nav bar-->
-    <div class="navA">
-      <img class="logo"
-        src="https://github.com/LambdaSchool/Sprint-Challenge--User-Interface/blob/master/img/lambda-black.png?raw=true"
-        alt="Lambda Company logo.">
-      <hearder>
-        <div class="nav-spacing">
-          <nav class="navB">
-            <a href="#">Home</a>
-            <a href="#">About</a>
-            <a href="#">Products</a>
-            <a href="#">Blog</a>
-            <a href="#">Contact</a>
-          </nav>
-        </div>
-    </div>
-
+<!--Nav bar-->
+  <div class="nav-a">
     <img class="logo"
-      src="https://github.com/LambdaSchool/Sprint-Challenge--User-Interface/blob/master/img/jumbo.jpg?raw=true"
-      alt="Header: About Image. Fast highway time elasped image">
-    </header>
-    <!-- Top Content starts here-->
+      src="https://github.com/LambdaSchool/Sprint-Challenge--User-Interface/blob/master/img/lambda-black.png?raw=true"
+      alt="Lambda Company logo.">
+    
+      <header>
+      <div class="nav-spacing">
+        <nav class="navB">
+          <a href="#">Home</a>
+          <a href="#">About</a>
+          <a href="#">Products</a>
+          <a href="#">Blog</a>
+          <a href="#">Contact</a>
+        </nav>
+      </div>
+  </div>
+  
+  <img class="logo1"
+    src="https://github.com/LambdaSchool/Sprint-Challenge--User-Interface/blob/master/img/jumbo-about.png?raw=true"
+    alt="Header: About Image. Skyline building view">
+  </header>
+
+  <!-- top content-->
+  <div class="container about-page">
+<p>
+    <div id="About-1">About</id>
+      <br>
+      <br>
+    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate
+    strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world
+    view of disruptive innovation via workplace diversity and empowerment.
+</div>
+</p>
+
+<!--ARTICLE1-->
+<div class="row1">
+  <article class= article1>
```

Comment:
In articles .. you chose to wrap an <img> and <p> tag as children elements.  Within the <p> tag,
you wrapped and <h2> header tag with plain text as well, and closed the <p> tag.  Semantically,
you should not wrap an <h2> tag WITHIN a <p> tag, as they have opposing semantic values.  A better
practice would be to have your <article> tag with a nested <img> tag, <h2> tag, and then <p> tag.
By pulling the <h2> outside of the <p> tag, the plain text now becomes wrapped in the <p> tag alone
and the <h2> tag gets its own semantic value within the <article>.